### ISSUE TITLE
Node 7.6 still a requirement?

### ISSUE THREAD

**Issue by GantMan** [infinitered/gluegun#335]
I know we had this built in for gluegun usage PRE-TypeScript:  https://github.com/infinitered/gluegun/blob/master/sniff.js

But now that we're compiling to common JS, can't we break the node >=7.6 requirement?

**Comment by jamonholmgren** [comment]
Yes, [but.](https://github.com/infinitered/gluegun/issues/306#issuecomment-354625392)

Essentially, we can support older versions of Node, yes. But the generated CLI will not work with older versions of Node unless we introduce a build step or significantly refactor.
